apps/system-e2e/src/support/email-account.ts

```diff
@@ -0,0 +1,154 @@
+// use Nodemailer to get an Ethereal email inbox
+// https://nodemailer.com/about/
+import { createTestAccount } from 'nodemailer'
+// used to check the email inbox
+import { connect } from 'imap-simple'
+// used to parse emails from the inbox
+// const simpleParser = require('mailparser').simpleParser
+import { simpleParser } from 'mailparser'
+import axios from 'axios'
+
+import {
+  GetIdentityVerificationAttributesCommand,
+  SESClient,
+  VerifyEmailAddressCommand,
+} from '@aws-sdk/client-ses'
+/**
+ * Register the email address with AWS SES so we can send emails to it
+ * @param emailAccount
+ */
+async function registerEmailAddressWithSES(emailAccount: {
+  getLastEmail(
+    retries: number,
+  ): Promise<{
+    subject: string | undefined
+    text: string | undefined
+    html: string | false
+  } | null>
+  email: string
+}) {
+  const client = new SESClient({ region: 'eu-west-1' })
+  await client.send(
+    new VerifyEmailAddressCommand({ EmailAddress: emailAccount.email }),
+  )
+  const verifyMsg = await emailAccount.getLastEmail(4)
+  if (verifyMsg && verifyMsg.text) {
+    console.log(`Verify message is ${verifyMsg.subject}: ${verifyMsg.text}`)
```

Comment:
Do we want the logger for this?

Reply:
This comment goes for the whole file

Reply:
I think this is mostly for troubleshooting problems while developing on a local instance. Will switch to our logging lib if we want this in DD at some point.